### PR TITLE
Refactored the kafka consumers to initialize the consuming function o…

### PR DESCRIPTION
**Pull request by AaronH88**
…nce the consumer has been created.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

**Review by Avielyo10 — APPROVED** [review]
lgtm

File: pkg/services/consumers.go [hunk 1]
```diff
@@ -72,6 +73,11 @@ func NewKafkaConsumerService(config *clowder.KafkaConfig, topic string) Consumer
 		shuttingDown:  false,
 		topic:         topic,
 	}
+	if topic == "platform.playbook-dispatcher.runs" {
+		s.consumer = s.ConsumePlaybookDispatcherRuns
+	} else if s.topic == "platform.inventory.events" {
+		s.consumer = s.ConsumeInventoryCreateEvents
+	}
```
Comment (ldjebran) [inline pkg/services/consumers.go:80]: if topic is not "platform.playbook-dispatcher.runs" and not "platform.inventory.events" what will happen?

**Review by ldjebran — COMMENTED** [review]
May be we are missing a case check

File: pkg/services/consumers.go (hunk 1 above)
Comment (AaronH88) [reply pkg/services/consumers.go:80]: Good spot! Ill fix now

**Pull request merged**